cmd/preflight/cmd/root.go

```diff
@@ -35,6 +35,8 @@ func rootCmd() *cobra.Command {
 	}
 
 	viper := viper.Instance()
+	rootCmd.PersistentFlags().String("config", "", "config file (default is config.yaml)")
```

Comment:
Please add the env equivalent to the docstring.

Reply:
@acornett21 done!